addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -722,11 +722,13 @@ def _import_fattura_pa(self, tree, invoice):
                     general_discount = discounted_amount - taxable_amount
                     sequence = len(elements) + 1
 
-                    with invoice_form.invoice_line_ids.new() as invoice_line_global_discount:
-                        invoice_line_global_discount.tax_ids.clear()
-                        invoice_line_global_discount.sequence = sequence
-                        invoice_line_global_discount.name = 'SCONTO' if general_discount < 0 else 'MAGGIORAZIONE'
-                        invoice_line_global_discount.price_unit = general_discount
+                    invoice_form.invoice_line_ids.create({
+                        'move_id': invoice_form.id,
+                        'tax_ids': [fields.Command.clear()],
+                        'name':'SCONTO' if general_discount < 0 else 'MAGGIORAZIONE',
+                        'sequence': sequence,
+                        'price_unit': general_discount
+                    })
```

Comment:
```suggestion
                    invoice_form.invoice_line_ids = [Command.create({
                        'move_id': invoice_form.id,
                        'tax_ids': [fields.Command.clear()],
                        'name':'SCONTO' if general_discount < 0 else 'MAGGIORAZIONE',
                        'sequence': sequence,
                        'price_unit': general_discount
                    })]
```

Reply:
Hey @smetl! My initial commit was in this format but @ThomasBeckers suggested using `create`.